bulbs/content/tasks.py

```diff
@@ -51,3 +52,93 @@ def update_feature_type_rates(featuretype_pk):
                 rate=0,
                 feature_type_id=featuretype_pk,
                 role_id=role.pk)
+
+
+@shared_task(default_retry_delay=5)
+def post_to_instant_articles_api(content_pk):
+    import requests
+    import logging
+
+    from django.conf import settings
+    from django.template.loader import render_to_string
+    from django.template.base import TemplateDoesNotExist
+
+    from bulbs.utils import vault
+    from bulbs.instant_articles.renderer import InstantArticleRenderer
+    from bulbs.instant_articles.transform import transform
+    from .models import Content
+
+    if getattr(settings, 'FACEBOOK_API_ENV', '').lower() == 'production':
```

Comment:
The API has a `development_mode` flag when posting, so maybe we should just set that with an environment variable. Part of the ticket was excluding this from local + testing, so I was just following that. @spra85 Any thoughts?